cmd/nerdctl/compose_exec_linux_test.go

```diff
@@ -165,7 +165,6 @@ services:
 func TestComposeExecTTY(t *testing.T) {
 	// `-i` in `compose run & exec` is only supported in compose v2.
 	// Currently CI is using compose v1.
```

Comment:
" Currently CI is using compose v1" is no longer true

Reply:
Thanks , it has changed. :-)